tcbuilder/backend/combine.py

```diff
@@ -21,6 +21,12 @@
     TARGET_NAME_FILENAME + ":/ostree/deploy/torizon/var/sota/storage/docker-compose/"
 ]
 
+DOCKER_FILES_TO_ADD_TO_RAW = [
+    "docker-compose.yml:/ostree/deploy/torizon/var/sota/storage/docker-compose/",
+    DOCKER_BUNDLE_FILENAME_UNCOMPRESSED + ":/ostree/deploy/torizon/var/lib/docker/:true",
+    TARGET_NAME_FILENAME + ":/ostree/deploy/torizon/var/sota/storage/docker-compose/"
+]
+
```

Comment:
I'd prefer to build variable `DOCKER_FILES_TO_ADD` dynamically (i.e. make it a local variable to the relevant function (`check_combine_files()`) and fill in its contents) based on whether compression is active or not instead of duplicating the whole variable.

Reply:
Resolving this since it was solved an a later commit in this PR.